launch-local-near-cluster.sh

```diff
@@ -0,0 +1,143 @@
+#  ${1} exists, evaluate ${@}"
+if ! "${KURTOSIS_CMD}" run --enclave-id "${ENCLAVE_ID}" "${NEAR_KURTOSIS_PACKAGE}" ${1+"${@}"} 2>&1 | tee "${exec_output_filepath}"; then
+    echo "Error: An error occurred executing package '${NEAR_KURTOSIS_PACKAGE}'" >&2
+    exit 1
+fi
+
+function get_json_property() {
+    package_output_filepath="${1:-}"
+    property_name="${2:-}"
+    if [ -z "${package_output_filepath}" ]; then
+        echo "Error: The filepath to the package output must be provided" >&2
+        return 1
+    fi
+    if [ -z "${property_name}" ]; then
+        echo "Error: A JSON property name must be provided" >&2
+        return 1
+    fi
+    cat "${package_output_filepath}" | grep "${property_name}" | awk '{print $NF}' | sed 's/^"//' | sed 's/",*$//'
+}
+
+validator_key_filepath="${package_exec_dirpath}/validator-key.json"
+if ! cat "${exec_output_filepath}" | awk "/${VALIDATOR_KEY_PROPERTY_W_QUOTES}/,/\}/" | sed "s/${VALIDATOR_KEY_PROPERTY_W_QUOTES}//" | sed 's/},/}/' > "${validator_key_filepath}"; then
+    echo "Error: Couldn't extract the validator key JSON from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! contract_helper_url="$(get_json_property "${exec_output_filepath}" "${HELPER_URL_PROPERTY}")"; then
+    echo "Error: Couldn't extract the contract helper URL from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! network_id="$(get_json_property "${exec_output_filepath}" "${NETWORK_ID_PROPERTY}")"; then
+    echo "Error: Couldn't extract the network ID from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! master_account="$(get_json_property "${exec_output_filepath}" "${MASTER_ACCOUNT_PROPERTY}")"; then
+    echo "Error: Couldn't extract the master account from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! node_url="$(get_json_property "${exec_output_filepath}" "${NODE_RPC_URL_PROPERTY}")"; then
+    echo "Error: Couldn't extract the NEAR node RPC URL from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! helper_url="$(get_json_property "${exec_output_filepath}" "${HELPER_URL_PROPERTY}")"; then
+    echo "Error: Couldn't extract the contract helper service URL from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! explorer_url="$(get_json_property "${exec_output_filepath}" "${EXPLORER_URL_PROPERTY}")"; then
+    echo "Error: Couldn't extract the explorer URL from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+if ! wallet_url="$(get_json_property "${exec_output_filepath}" "${WALLET_URL_PROPERTY}")"; then
+    echo "Error: Couldn't extract the wallet URL from package exec logfile '${exec_output_filepath}'" >&2
+    exit 1
+fi
+
+echo "============================================================ SUCCESS ================================================================================"
```

Comment:
Wondering what this is about. I've never used that thing before

Reply:
https://docs.near.org/develop/testing/kurtosis-localnet this is how they encourage using it.